attributes/default.rb

```diff
@@ -1,3 +1,4 @@
 default['build-essential']['compile_time'] = true
 
 default['jlaws']['aws_sdk_ver'] = '3.0.1'
+default['jlaws']['aws_sdk_ver'] = '3.18.0'
```

Comment:
Is `aws-sdk-resources` version synced with `aws-sdk`?

Reply:
apparently not 
https://rubygems.org/gems/aws-sdk/versions/3.0.1